docs/sources/operators-guide/configure/configure-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/configure-helm-ha-deduplication-consul/
+description: Learn how to configure Grafana Mimir Helm Chart to handle HA Prometheus server deduplication with Consul.
+menuTitle: Configure high-availability deduplication with Consul
+title: Configuring Grafana Mimir Helm Chart for high-availability deduplication with Consul
+weight: 70
+---
+
+# Configure high-availability deduplication with Consul
+
+Grafana Mimir can deduplicate data from a high-availability (HA) Prometheus setup. You can configure
+the deduplication by using the Grafana Mimir helm chart deployment that uses external Consul. For more information, see [Configure high availability]({{< relref "../configure-high-availability-deduplication.md" >}}).
+
+## Before you begin
+
+You need to have a Grafana Mimir installed.
+
+For conceptual information about how Mimir deduplicates incoming HA samples, refer to [Configure high availability]({{< relref "../configure-high-availability-deduplication.md" >}}).
+
+You also have to configure HA for Prometheus or Grafana Agent. Last, you need a KV store. In this guide you will
+use Consul. You will be guided on the setup if you haven't had one.
+
+## Configure Prometheus or Grafana Agent to send HA external labels
+
+Configure Prometheus or Grafana Agent HA setup by setting label called `cluster` and `__replica__`. These two labels
+name are default labels for HA setup in Grafana Mimir. If you want to change the labels, make sure to change it in Mimir
+too to make Grafana Mimir and Prometheus/Grafana Agent config matches with each other, otherwise HA deduplication
+wouldn't work. `cluster` label value must be same across replica belong to the same cluster. `__replica__` must be
+unique across different replica in the cluster.
```

Comment:
```suggestion
Configure the Prometheus or Grafana Agent HA setup by setting the labels named `cluster` and `__replica__`,
which are the default labels for a HA setup in Grafana Mimir. If you want to change a label,
make sure to change it in Mimir as well. This ensures that the configurations of Grafana Mimir, Prometheus, and Grafana Agent all match each other. Otherwise, HA deduplication will not work.
 * The value of the `cluster` label must be same across replica that belong to the same cluster.
 * The value of the `__replica__` label must be unique across different replica within the same cluster.
```

Reply:
Please vet this for technical accuracy.

Reply:
For technical accuracy, the second sentence should be something like

```
If you want to change the HA labels, make sure to change them in Mimir as well.
```

instead of 

```
If you want to change a label, make sure to change it in Mimir as well.
```

because we are talking about specific label pairs with the default name `cluster` and `__replica__`.

Please review it again.